### ISSUE TITLE
pictures/media sent from signal-cli-rest-api don't show up on the Signal iOS app (v. 7.26 and newer)

### ISSUE THREAD

**Issue by GatoPharaoh** [bbernhard/signal-cli-rest-api#583]
### The problem

I noticed that pictures, videos and audio sent from signal-cli-rest-api don't show up on the iOS app anymore. doing a little testing i found out that this issue started with the **introduction of Signal iOS version 7.26**.

pictures/media do show up in paired clients for example on desktop, but not on any iOS client (be it native or paired) that has an app version of 7.26 or later (Signal iOS version **7.25 Build 275** is the last that i found receiving media). text messages do still work and are received but as soon as a picture/media file is involved, the iOS app does not show any message.

as sent pictures and media from other iOS and android devies are still received, is this maybe an issue of "how" the pictures are sent by signal-cli-rest-api? is anyone else experiencing this issue?

### Are you using the latest released version?

- [X] Yes

### Have you read the troubleshooting page?

- [X] Yes

### What type of installation are you running?

signal-cli-rest-api Docker Container

### In which mode are you using the docker container?

JSON-RPC Mode

### What's the architecture of your host system?

x86-64

**Comment by steveredden** [comment]
Same issue for me. Even the simplest base64 is not sent. getting `201` response codes, though...

I tested upgrading my container to use [signal-cli:v0.13.6](https://github.com/AsamK/signal-cli/releases/tag/v0.13.6) and **was able** to get media to send.

1. Get container id:  `docker ps | grep signal`
2. `docker exec -it <id from step 1> /bin/bash`
3. `apt update`
4. `apt install wget`
5. `export VERSION=0.13.6`
6. `wget https://github.com/AsamK/signal-cli/releases/download/v"${VERSION}"/signal-cli-"${VERSION}".tar.gz`
7. `tar xf signal-cli-"${VERSION}".tar.gz -C /opt`
8. `ln -sf /opt/signal-cli-"${VERSION}"/bin/signal-cli /usr/local/bin/`

**Comment by bbernhard** [comment]
Please give this version a try: `bbernhard/signal-cli-rest-api:0.167-dev`

**Comment by steveredden** [comment]
that WORKED! I can successfully send b64 attachment and photo renders on my signal app.

but... the API returned a 400 (odd):  `[GIN] 2024/09/13 - 14:22:02 | 400 |  2.139253668s |      172.19.0.1 | POST     "/v2/send"`

```
"{"error":"1726237322665SLF4J(I): Connected with provider of type [ch.qos.logback.classic.spi.LogbackServiceProvider]"}"
```

**Comment by bbernhard** [comment]
really strange - that must come from signal-cli. Which mode are you using?

**Comment by steveredden** [comment]
MODE=native

**Comment by bbernhard** [comment]
looks like this is already fixed upstream, see: https://github.com/AsamK/signal-cli/issues/1584

**Comment by GatoPharaoh** [comment]
> Please give this version a try: `bbernhard/signal-cli-rest-api:0.167-dev`

did a quick test, seems to work! thank you!

**Comment by bbernhard** [comment]
> looks like this is already fixed upstream, see: [AsamK/signal-cli#1584](https://github.com/AsamK/signal-cli/issues/1584). Hope there will be another release soon

Should be fixed with `bbernhard/signal-cli-rest-api:0.168-dev` - which is a potential release candidate. If no regressions pop up during testing, this version will be officially released soon. Any help with testing is greatly appreciated!

**Comment by bbernhard** [comment]
Should be fixed with the latest release
